docs/user/dql/functions.rst

```diff
@@ -1219,15 +1219,60 @@ Example::
     +---------------------------+
 
 
+MAKEDATE
+--------
+
+Description
+>>>>>>>>>>>
+
+Returns a date, given `year` and `day-of-year` values. `dayofyear` must be greater than 0 or the result is `NULL`. The result is also `NULL` if either argument is `NULL`.
+Arguments are rounded to an integer.
+
+Specifications:
+
+1. MAKEDATE(DOUBLE, DOUBLE) -> DATE
+
+Argument type: DOUBLE
+
+Return type: DATE
+
+Example::
+
+    os> select MAKEDATE(1945, 5.9) as f1, MAKEDATE(1984, 1984) as f2
+    fetched rows / total rows = 1/1
+    +------------+------------+
+    | f1         | f2         |
+    |------------+------------|
+    | 1945-01-06 | 1989-06-06 |
+    +------------+------------+
+
+
 MAKETIME
 --------
 
 Description
 >>>>>>>>>>>
 
+Returns a time value calculated from the hour, minute, and second arguments. Returns `NULL` if any of its arguments are `NULL`.
+The second argument can have a fractional part, rest arguments are rounded to an integer.
```

Comment:
Is the fractional part rounded to a particular number of decimal places?

Reply:
What about hour > 24?  What about minute/second > 60? 
Make sure to indicate 24 hour time is used, not 12-hour with am/pm.